### PR TITLE
Removing old changes

### PR DESCRIPTION
**Pull request by sunlnu**
# stream-registry PR

_&lt;Removing all code which is related to renaming and was added in version `1.1.3` and `1.3.0`&gt;_

### Deleted
* _&lt;Removing all code which is related to renaming and was added in version `1.1.3` and `1.3.0`&gt;_

# PR Checklist Forms

- [ ] CHANGELOG.md updated
- [ ] Reviewer assigned
- [ ] PR assigned (presumably to submitter)
- [ ] Labels added (enhancement, bug, documentation)

### CODE REVIEW

File: state/core/src/test/java/com/expediagroup/streamplatform/streamregistry/state/DefaultEntityViewUpdaterTest.java [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package com.expediagroup.streamplatform.streamregistry.state;
 
 import com.expediagroup.streamplatform.streamregistry.state.model.Entity;
 
 import java.util.Map;
 
+import com.expediagroup.streamplatform.streamregistry.state.model.Entity;
```
Comment (techabstraction) [inline state/core/src/test/java/com/expediagroup/streamplatform/streamregistry/state/DefaultEntityViewUpdaterTest.java:22]: duplicate import?

**Review by techabstraction — COMMENTED** [review]
Is it intentional that not all of the changes in this PR are being reverted? 🤔  https://github.com/ExpediaGroup/stream-registry/pull/325

See `GraphQLConverter.java` for example.

**Comment by sunlnu** [comment]
> Is it intentional that not all of the changes in this PR are being reverted? 🤔 #325
> 
> See `GraphQLConverter.java` for example.

Hey @techabstraction  thanks for reviewing , I checked and i found version 1.1.3 having issue which is this PR  [1.1.3](https://github.com/ExpediaGroup/stream-registry/commit/64ebe6c16a20dc8e357a7849129b6d43a7943375#diff-6e2a11bc476cd096fbb11477314e70fe7214cf80c4ccda8612ffd69fc9a24884)
And at time i created 2 prs one is related to domain check before creating process and it was in version 1.1.2  and that our application using but after you discussed with me and asked me to change some variables names So i created one more PR which 1.1.3 but i guess there was some issue and version didn't create but code was merged and in later branches it come as a issue .
So, i am removing the variable change related code only rest is working fine as check in local machine .
Your change is in #325 is working fine as checked in local

File: state/core/src/test/java/com/expediagroup/streamplatform/streamregistry/state/DefaultEntityViewUpdaterTest.java (hunk 1 above)
Comment (sunlnu) [reply state/core/src/test/java/com/expediagroup/streamplatform/streamregistry/state/DefaultEntityViewUpdaterTest.java]: changed

**Comment by sunlnu** [comment]
we will add these variable changes again once our 1.3.1 works fine in egsp-stream-reg and SDK

**Comment by techabstraction** [comment]
I think we should undo all the changes that were made to renaming the zone and stream key fields, not just the changes from the first pr

**Comment by sunlnu** [comment]
@techabstraction can you check once more , i moved back to earlier naming .

**Review by techabstraction — APPROVED** [review]

**Comment by jamespfaulkner** [comment]
Should we update the changelog to indicate that all changes in `1.1.3` have been reverted?

**Review by jamespfaulkner — APPROVED** [review]

**Pull request merged**